source/MRMesh/MRClosestWeightedPoint.h

```diff
@@ -59,11 +88,12 @@ struct DistanceFromWeightedPointsParams
 
 struct DistanceFromWeightedPointsComputeParams : DistanceFromWeightedPointsParams
 {
-    /// stop searching as soon as any point within this distance is found
-    float minDistance = -FLT_MAX; // default 0 here does not work for negative distances
+    /// stop searching as soon as any point within this weighted distance is found,
+    /// ignored if bidirectionalMode=false, because in one-directional mode, weighted distance approximation not only decreases
+    float minBidirDist = -FLT_MAX; // default 0 here does not work for negative distances
 
-    /// find the closest point only if the distance to it is less than given value
-    float maxDistance = FLT_MAX;
+    /// find the closest point only if weighted bidirectional distance to it is less than given value
+    float maxBidirDist = FLT_MAX;
```

Comment:
does it work in `bidirectionalMode=false`?

Reply:
Yes, it is always used to set the maximum search radius
```
maxSearchRadius_( params.maxBidirDist + params.maxWeight )
```